src/core/hooks/CodeBlock.js

```diff
@@ -203,7 +203,7 @@ export default class CodeBlock extends ParagraphBase {
    */
   $getIndentedCodeReg() {
     const ret = {
-      begin: '(?:^|\\n\\s*\\n)(?: {4}|\\t)',
+      begin: '(?<=^|\\n)(?: {4}|\\t)',
```

Comment:
Avoid using lookbehind assertion in regexp. Because not all morden browsers support it, such as Safari.

FYI: https://caniuse.com/?search=lookbehind

Reply:
use `?:` and `prependLineFeedForParagraph` instead.